GitHeart/MemoryCache/MemoryCache.swift

```diff
@@ -15,6 +15,7 @@ protocol ByteSizable {
 
 /// A structure to cache values by a key, stores data in memory.
 ///
+// Does this type of cache has a common name? I heard about LRU, but not sure it it's used in here.
```

Comment:
This is not exactly LRU cache because it doesn't update position of a value towards more recent values when we access it using `value(forKey:)` I guess I can implement that.